screens/EventDetails/EventDetails.tsx

```diff
@@ -0,0 +1,258 @@
+                      size={6}
+                      color={ButtonStyles.colorOpacity35}
+                      name="ellipse"
+                    />
+                    <Headline onPress={sendFriendRequest} style={{ color: event.color }}>Add Friend</Headline>
+                  </View>
+                )
+                : (
+                  <View style={[styles.flexRow, { alignItems: "center" }]}>
+                    <Ionicons
+                      style={{ marginHorizontal: 8 }}
+                      color={ButtonStyles.colorOpacity35}
+                      size={6}
+                      name="ellipse"
+                    />
+                    <Headline onPress={sendFriendRequest} style={{ color: ButtonStyles.colorOpacity35 }}>Request Sent</Headline>
+                  </View>
+                )
+              }
+              
+            </View>
+            <Caption style={styles.textColor}>{event.host.handle}</Caption>
+          </View>
+        </View>
+
+        <View style={styles.iconSection}>
+          <TimeSection color={event.color} event={calenderEvent} />
+
+          <Divider style={styles.divider} />
+
+          <LocationSection
+            color={event.color}
+            coordinates={event.coordinates}
+            placemark={event.placemark}
+            bottomTabHeight={BOTTOM_TAB_HEIGHT}
+          />
+
+          <Divider style={styles.divider} />
+
+          <AttendeeSection
+            color={event.color}
+            attendeeCount={event.attendeeCount}
+          />
+
+          <Divider style={styles.divider} />
+
+          <ChatSection
+            color={event.color}
+            userAttendeeStatus={event.userAttendeeStatus}
+          />
+        </View>
+
+        {event.description && (
+          <View style={{ marginTop: 16 }}>
+            <Headline style={[styles.textColor, {marginBottom: 4 }]}>About</Headline>
+            <ExpandableText
+              props={{ style: { color: event.color, marginTop: 5 } }}
+              text={event.description}
+              linesToDisplay={3}
+            />
+          </View>
+        )}
+        <View style={{ marginTop: 16, marginBottom: BOTTOM_TAB_HEIGHT + 24 }}>
+          <Headline style={[styles.textColor, { marginBottom: 8 }]}>Location</Headline>
+          <TouchableOpacity
+            style={{
+              borderRadius: 12,
+              overflow: "hidden"
+            }}
+            onPress={openMapWithoutDirections}
+          >
+            <EventMapSnippet
+              style={{ width: "100%", height: 160 }}
+              minZoomLevel={12}
+              initialRegion={{
+                latitude: event.coordinates.latitude,
+                longitude: event.coordinates.longitude,
+                latitudeDelta: 0.1,
+                longitudeDelta: 0.1
+              }}
+              renderMarker={() => (
+                <Image
+                  source={require("../../assets/icon.png")}
```

Comment:
I think @assets will act as an absolute path, though I haven't checked in a while